### PR TITLE
Fix #2777. Kill idle docker and local agents

### LINKED ISSUE #2777: Idle timeout of local agents
We currently only timeout docker drivers. We should do this for docker and local drivers.

### PR DESCRIPTION
**Pull request by e-carlin**

### CODE REVIEW

File: sirepo/job_driver/__init__.py [hunk 1]
```diff
@@ -353,6 +370,7 @@ def init(job_supervisor_module):
     job_supervisor = job_supervisor_module
     cfg = pkconfig.init(
         modules=((_DEFAULT_MODULE,), set, 'available job driver modules'),
+        idle_check_secs=(1800, int, 'how many seconds to wait between checks'),
```
Comment (robnagler) [inline sirepo/job_driver/__init__.py:373]: parse_seconds. reuse name: idle_task_secs?

File: sirepo/job_driver/__init__.py [hunk 2]
```diff
@@ -310,6 +311,7 @@ def _receive_alive(self, msg):
 
         Save the websocket and register self with the websocket
         """
+        self._start_idle_timeout()
```
Comment (robnagler) [inline sirepo/job_driver/__init__.py:314]: I would put the start_idle_timeout inside the agent starting done.

File: sirepo/job_driver/__init__.py (hunk 1 above)
Comment (e-carlin) [reply sirepo/job_driver/__init__.py]: parse_seconds  - https://github.com/radiasoft/sirepo/pull/2804/commits/697d75b03e5b4d2b980c6307ba4ed4910673df0d

What do you mean by "reuse name". Don't reuse the name from job_driver.docker?

File: sirepo/job_driver/__init__.py (hunk 2 above)
Comment (e-carlin) [reply sirepo/job_driver/__init__.py]: https://github.com/radiasoft/sirepo/pull/2804/commits/697d75b03e5b4d2b980c6307ba4ed4910673df0d

**Pull request merged**